src/App.js

```diff
@@ -33,78 +32,75 @@ class App extends Component {
     // slice(1) removes the `?` at the beginning of `location.search`
     const infoMessage = qs.parse(location.search.slice(1)).infoMessage;
     return (
-      <div className="application-content">
-        <Metadata />
-        <Spinner name="apiSpinner">
-          <div className="loader-bg">
-            <div className="loader" />
-          </div>
-        </Spinner>
-        <Header
-          language={language}
-          changeLanguage={changeLanguage}
-          message={infoMessage}
-        />
-        <Switch>
-          <Route
-            path="/:lang?/store"
-            render={matchProps => (
-              <StorePage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            path="/:lang?/coffees/:coffeeSlug"
-            render={matchProps => (
-              <CoffeePage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            path="/:lang?/brewers/:brewerSlug"
-            render={matchProps => (
-              <BrewerPage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            exact
-            path="/:lang?/articles"
-            render={() => <ArticlesPage language={language} />}
-          />
-          <Route
-            path="/:lang?/articles/:articleId"
-            render={matchProps => (
-              <ArticlePage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            path="/:lang?/cafes"
-            render={() => <CafesPage language={language} />}
-          />
-          <Route
-            path="/:lang?/contacts"
-            render={() => <ContactsPage language={language} />}
-          />
-          <Route
-            exact
-            path="/:lang?"
-            render={matchProps => (
-              <HomePage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            path="/:lang?/:urlSlug?"
-            render={matchProps => (
-              <AboutPage {...matchProps} language={language} />
-            )}
-          />
-          <Route
-            path="*"
-            render={() => {
-              return <Redirect to="/" push />;
-            }}
+      <SpinnerLayout>
```

Comment:
Do you also need it here? As I see it the spinner service is only in the configuration component

Reply:
It is being used in Stores https://github.com/Kentico/kontent-sample-app-react/search?q=spinnerService (soon téo be in page components #201)